src/main/webapp/WEB-INF/tags/instructor/course/courseEditAccessControlEditDivForInstr.tag

```diff
@@ -8,10 +8,7 @@
 <div id="accessControlEditDivForInstr${instructorPanel.index}">
     <c:if test="${instructorPanel.accessControlDisplayed}">
         <div class="form-group">
-            <div class="col-sm-3">
-                <label class="control-label pull-right">Access-level</label>
-            </div>
-
```

Comment:
This removal is also unrelated - there can be a blank line; just make sure there are no trailing spaces.